Document MS Teams integration (#183)
spacelift-io/user-documentation: docs/integrations/chatops/msteams.md

---
description: Integrating Spacelift with Microsoft Teams
---

# Microsoft Teams

> [Microsoft Teams](https://www.microsoft.com/en-us/microsoft-teams/group-chat-software) is a Slack alternative and a part of the Microsoft Office 365 suite. It's a chat-based workspace in which teams can organize and discuss their work. Many DevOps teams use it to communicate and collaborate on infrastructure and application deployments. Hence, Spacelift has a first class integration with Microsoft Teams.

Reviewer: Per Gosia's request and for SEO reasons, all non-Spacelift links must have the `nofollow` relation.

```suggestion
[Microsoft Teams](https://www.microsoft.com/en-us/microsoft-teams/group-chat-software){: rel="nofollow"} is a Slack alternative and a part of the Microsoft Office 365 suite. It's a chat-based workspace where teams can organize and discuss their work. Many DevOps teams use it to communicate and collaborate on infrastructure and application deployments. Hence, Spacelift has a first-class integration with Microsoft Teams.
```
